reaktive/src/commonTest/kotlin/com/badoo/reaktive/maybe/OnErrorResumeNextTest.kt

```diff
@@ -119,11 +120,16 @@ class OnErrorResumeNextTest :
 
     @Test
     fun produces_error_WHEN_upstream_produced_error_and_resume_next_supplier_produced_error() {
-        val throwable = Throwable()
-        val observer = upstream.onErrorResumeNext { throw throwable }.test()
-        upstream.onError(Throwable())
-
-        observer.assertError(throwable)
+        val upstreamThrowable = Throwable()
+        val supplierThrowable = Throwable()
+        val observer = upstream.onErrorResumeNext { throw supplierThrowable }.test()
+        upstream.onError(upstreamThrowable)
+
+        observer.assertError { throwable ->
+            throwable is CompositeException &&
+                    throwable.cause1 == upstreamThrowable &&
```

Comment:
We are following Kotlin code styles, continuation indent should be 4 spaces, not 8.
We will consider to fix it in the repo somehow, but please change it manually in IDEA settings and reformat all the code.
![image](https://user-images.githubusercontent.com/26204457/62329436-c8a35500-b4ad-11e9-93d0-e16a0f4de36f.png)

Reply:
Fixed